File: app/src/main/java/chenmc/sms/transaction/EmptyService.java
package chenmc.sms.transaction;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.support.annotation.Nullable;

/**
 * Created by 明明 on 2017/7/21.
 */

public class EmptyService extends Service {
    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }
    
    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        stopSelf();
        return START_NOT_STICKY;
    }
}

File: app/src/main/java/chenmc/sms/transaction/service/EmptyService.kt
package chenmc.sms.transaction.service

import android.app.Service
import android.content.Intent
import android.os.IBinder

/**
 * Created by 明明 on 2017/7/21.
 */

class EmptyService : Service() {

    override fun onBind(intent: Intent): IBinder? {
        return null
    }

    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
        stopSelf(startId)
        return Service.START_NOT_STICKY
    }
}
